### PR TITLE
BUG: fix to_postgis appending to table without CRS

### PR DESCRIPTION
**Pull request by claushofmann**
Fixes bug mentioned in #2158

### CODE REVIEW

File: geopandas/io/sql.py [hunk 1]
```diff
@@ -417,6 +417,10 @@ def _write_postgis(
                     )
                 ).fetchone()[0]
 
+                # Postgis uses SRID 0 to signify the absence of CRS, we use -1
+                if target_srid == 0:
+                    target_srid = -1
```
[the remark this answers is not in the document]
Comment (claushofmann) [reply geopandas/io/sql.py:422]: Hi,
Unfortunately I don't know if there is a specific reason. I just wanted to fix this this single bug and figured this was the safest way which doesnt risk breaking anything else somewhere on the code.

**Comment by claushofmann** [comment]
Weird... I don't really understand why `Tests / ubuntu-latest, ci/envs/37-pd10.yaml (pull_request)` failed. Is it common that this test has a timeout?

**Comment by martinfleis** [comment]
>  Is it common that this test has a timeout?

I think I've seen it couple of times before. Not common, but it happens (we should check why).

**Comment by jorisvandenbossche** [comment]
Closing as the follow-up in https://github.com/geopandas/geopandas/pull/3328 has been merged. @claushofmann thanks for the original PR, and apologies for the slow follow-up !

**Pull request closed without merging**